src/vs-bicep/Bicep.VSLanguageServerClient.ItemTemplate/Bicep.VSLanguageServerClient.ItemTemplate.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>17.0</MinimumVisualStudioVersion>
```

Comment:
Does this need to be 17.3, or not related?

Reply:
Yes, I logged bug for that https://github.com/Azure/bicep/issues/7911

I'll update it after 17.3 GA and when the build machines are updated to use that version of VS